src/plugins/intel_gpu/src/runtime/CMakeLists.txt

```diff
@@ -29,8 +29,7 @@ endif()
 
 target_include_directories(${TARGET_NAME} PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
-    $<BUILD_INTERFACE:${INCLUDE_DIR}>
-    $<BUILD_INTERFACE:${MAIN_DIR}/src/graph/include>)
+    $<BUILD_INTERFACE:${INCLUDE_DIR}>)
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017858297
Reverted. Thank you.